Events/src/main/java/com/github/swcraftlyon/meetup/infra/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.github.swcraftlyon.meetup.infra;
+
+import com.github.swcraftlyon.meetup.domain.IPlanner;
+import com.github.swcraftlyon.meetup.domain.entities.Event;
+import com.github.swcraftlyon.meetup.domain.services.EventManager;
+import com.github.swcraftlyon.meetup.domain.services.IEventManager;
+import com.github.swcraftlyon.meetup.infra.adapters.repositories.PlannerRepository;
+import com.github.swcraftlyon.meetup.infra.dao.IEventDAO;
+import com.github.swcraftlyon.meetup.infra.dao.MangoEventDao;
+
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
The answer will be interesting, a java program needs a main to be launched, unless we use a library or framework that has a hidden main.